### ISSUE TITLE
issue with tags:

### ISSUE THREAD

**Issue by steverweber** [ansible/ansible#2500]
[15:01] <s8weber> somthing seems strange with list tasks
[15:01] <s8weber> ansible-playbook tags.yml -K --tags=foo --list-tasks
[15:01] <s8weber> gives error
[15:01] <s8weber> ERROR: tag(s) not found in playbook: foo.  possible values: bar,base
[15:03] <s8weber> also if you change tags.yml first tag to a oneliner version tags: extra
[15:03] <s8weber> you get this
[15:03] <s8weber> ERROR: tag(s) not found in playbook: fo.  possible values: a,e,foo,r,t,x
[15:03] <s8weber> when running :  ansible-playbook tags.yml -K --tags=fo --list-tasks

**Comment by mpdehaan** [comment]
Can you please post a playbook that reproduces this issue?  If multiple files, gist.github.com may be a good choice.

Thanks!

**Comment by steverweber** [comment]
this is the tweaked example tags.yml... 
tags.yml its included with the Ansible source example/ 

```
# tags allow us to run all of a playbook or part of it.
#
# assume: ansible-playbook tags.yml --tags foo
#
# try this with:
#    --tags foo
#    --tags bar
#    --tags extra
#
# the value of a 'tags:' element can be a string or list
# of tag names.  Variables are not usable in tag names.


- name: example play one
  hosts: all
  user: root

  # any tags applied to the play are shorthand to applying
  # the tag to all tasks in it.  Here, each task is given
  # the tag extra

  tags: extra

  tasks:

  # this task will run if you don't specify any tags,
  # if you specify 'foo' or if you specify 'extra'

  - name: hi
    tags: foo
    action: shell echo "first task ran"

- name: example play two
  hosts: all
  user: root
  tasks:
  - name: hi
    tags:
       - bar
    action: shell echo "second task ran"
  - include: tasks/base.yml tags=base
```

**Comment by mpdehaan** [comment]
Fixed per e418d2fe6235cf182f7f4543ac47c3f3b511aaeb
